src/main/java6/net/finmath/marketdata/model/curves/ForwardCurve.java

```diff
@@ -73,38 +77,17 @@ public ForwardCurve(String name, LocalDate referenceDate, String paymentOffsetCo
 	}
 
 	/**
-	 * Generate a forward curve using a given discount curve and payment offset. The forward F(t) of an index is such that
-	 * F(t) * D(t+p) equals the market price of the corresponding index fixed in t and paid in t+d, where t is the fixing time
-	 * of the index and t+p is the payment time of the index. F(t) is the corresponding forward and D is the associated discount
-	 * curve.
-	 * 
-	 * @param name The name of this curve.
-	 * @param referenceDate The reference date for this code, i.e., the date which defines t=0.
-	 * @param paymentOffsetCode The maturity of the index modeled by this curve.
-	 * @param interpolationEntityForward Interpolation entity used for forward rate interpolation.
-	 * @param discountCurveName The name of a discount curve associated with this index (associated with it's funding or collateralization), if any.
+	 * See main constructor

```

Comment:
What does "See main constructor" mean here? Which constructor is the main constructor? Please remove this line or use a JavaDoc @see pointing to the exact constructor (but it is still not clear "what to see there". Also beware that "the main constructor" could change and the programmer may not be aware of this reference.